elf/input-files.cc

```diff
@@ -324,7 +324,6 @@ void ObjectFile<E>::initialize_ehframe_sections(Context<E> &ctx) {
     std::unique_ptr<InputSection<E>> &isec = sections[i];
     if (isec && isec->is_alive && isec->name() == ".eh_frame") {
       read_ehframe(ctx, *isec);
-      isec->is_alive = false;
```

Comment:
If a section is not copied from input files to an output file as-is, I believe we mark it as dead. So we want to keep this code.

Reply:
I switched to an approach that is similar to what is used for mergeable sections, i.e. killing the sections after symbol resolution.